util/imagetools/loader.go

```diff
@@ -283,7 +284,9 @@ func (l *loader) scanSBOM(ctx context.Context, fetcher remotes.Fetcher, r *resul
 				return nil, errors.Errorf("referenced image %s not found", dgst)
 			}
 			for _, layer := range mfst.manifest.Layers {
-				if layer.MediaType == "application/vnd.in-toto+json" && layer.Annotations["in-toto.io/predicate-type"] == "https://spdx.dev/Document" {
+				if (layer.MediaType == "application/vnd.in-toto+json" ||
+					layer.MediaType == "application/vnd.in-toto.spdx+dsse") &&
```

Comment:
Do you have an image we can look at with this mime type?

Reply:
[docker.io/jamescarnegie559/kms-test:latest](http://docker.io/jamescarnegie559/kms-test:latest)